lib/Conversion/AIEVecToLLVM/AIEVecToLLVM.cpp

```diff
@@ -1584,9 +1584,10 @@ class MaxOpConversion : public mlir::ConvertOpToLLVMPattern<aievec::MaxOp> {
     // create xllvm intrinsic
     Value maxOp = nullptr;
     if (llvm::isa<IntegerType>(resultScaTy)) {
-      // create constant for cmp
+      // create constant for third operand `cmp`
+      // Note: `cmp` is implicitly treated as `sign` to the vmax intrinsic
```

Comment:
Thanks for the comment.
This is an oversight in the definition of aievec.min and aievec.max. The existing testcases assume signed comparison.
These should have an attribute specifying whether it uses a signed comparison or an unsigned comparison.
In the arith dialect, there are 4 min ops: minsi, minui, mininumF, minNumF. We deliberately wanted a single op to handle both the integer and floating point cases, but we failed to distinguish between minsi and minui.
Regarding minimumF vs minNumF, I did not find in the arch doc whether -0 < +0.